File: bank-issuer/src/main/java/com/project/bankissuer/dto/PaymentRequestDto.java
package com.project.bankissuer.dto;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class PaymentRequestDto {
    private String pan;
    private String securityCode;
    private String cardholderName;
    private String expirationMonth;
    private String expirationYear;
    private String paymentId;
    private Double amount;

    private String acquirerOrderId;
    private LocalDateTime acquirerTimestamp;

    private String qr;
}

File: bank-acquirer-front/src/app/dto/payment-request-dto.ts
export class PaymentRequestDto{
  constructor(
    public pan:string='',
    public securityCode:string='',
    public cardholderName:string='',
    public expirationMonth:string='',
    public expirationYear:string='',
    public paymentId:string='',
  ) {}
}
